insecure/integration/test/composability_test.go

```diff
@@ -48,6 +48,11 @@ func TestCorruptibleConduitFrameworkHappyPath(t *testing.T) {
 				// implementing the corruption functionality of the orchestrator.
 				event.FlowProtocolEvent = corruptedEvent
 			}, func(t *testing.T) {
+
+				require.Eventually(t, func() bool {
+					return ccf.AttackerRegistered() // attacker's registration must be done on CCF prior to sending any messages.
+				}, 2*time.Second, 100*time.Millisecond, "registration of attacker on CCF could not be done one time")
```

Comment:
we're giving 2 second limit for registering attacker? could we guarantee a lower time for registration?